utils/ogmios-client/src/query_network.rs

```diff
@@ -0,0 +1,33 @@
+//! Queries that start with `queryNetwork/`.
+
+use crate::{types::SlotLength, OgmiosClient, OgmiosClientError, OgmiosParams};
+use serde::Deserialize;
+use serde_json::Value;
+use std::collections::HashMap;
+
+pub trait QueryNetwork: OgmiosClient {
+	#[allow(async_fn_in_trait)]
+	async fn shelley_genesis_configuration(
```

Comment:
Would it make any sense to already make this generic instead of hardcoding shelley? It would be sufficient to just introduce an `era` argument wouldn't it?

Reply:
No, because responses are different. If `era` becomes a parameter, then response cannot be that specific. I would say json Value.